ts/Core/Axis/AxisDefaults.ts

```diff
@@ -232,7 +232,7 @@ namespace AxisDefaults {
          * @sample {highcharts} highcharts/xaxis/crosshair-both/
          *         Crosshair on both axes
          * @sample {highstock} stock/xaxis/crosshairs-xy/
-         *         Crosshair on both axes
+         *         Crosshair on both axes, with yAxis label
```

Comment:
Code must be either be written in backticks, `yAxis` or in plain English: y axis. I think plain English works best in this case.

Reply:
Corrected, thank you!